bin/eve.js

```diff
@@ -50,6 +51,10 @@ if(argv["help"]) {
     --port <number> Change the port the Eve server listens to (default 8080).
     --localControl  Entirely disable server interaction. File changes will be
                     stored in localStorage.
+    --json          Load the passed json file into @init. One additional
+                    colon separated argument to specify the target database,
+                    an additional to add the passed tag to each json record
+                    [[tag:]database]:filename
```

Comment:
well, it clearly doesn't matter semantically. i guess i was thinking kinda like the database is the containing object? the tag is kinda the same? i'll flip it unless you think we should get a third opinion